### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kcoder63**
CONTRIBUTING.md for heath check

Closes PR #34

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+# 1. Introduction
+**Welcome!** Thank you for contributing to the further development of Health Check. We're always looking for new ways to improve our project and we appreciate any help you can give us.
+
+
+### 1.1 What kinds of contributions are we looking for?
+We love receiving contributions from our community. Any assistance you can provide with regards to bug fixes, feature enhancements, and documentation is more than welcome.
+
+# 2. Ground Rules
+1. Ensure cross compatibility for Windows, Mac OS and Linux.
+2. Ensure all Python features used in contributions exist and work in Python 3.8.1 and above.
```
Comment (kritikaparmar-programmer) [inline CONTRIBUTING.md:23]: Make this Python 3.7 or greater. We can use Python 3.7 for this project

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+* help-wanted - issues that are currently unassigned to anyone and may be a bit more involved/complex than issues tagged with beginner.
+* Feel free to ask for help; everyone was a beginner at some point!
+
+# 4. Getting Started
+
+Edbeepbot's repository is configured to follow a particular development workflow, using various reputable tools. We kindly ask that you stick to this workflow when contributing to Edbeepbot, by following the guides below. This will help you to easily produce quality code, identify errors early, and streamline the code review process.
+
+### 4.1 Setting up your development environment
+The following requirements must be installed prior to setting up:
+ - Python `3.8.1` or greater
```
Comment (kritikaparmar-programmer) [inline CONTRIBUTING.md:40]: Here also make it Python 3.7

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    ```
+    git pull upstream master
+    ```
+* Create your feature branch
+    ```
+    git checkout -b <feature-name>
+    ```
+* Commit all the changes
+    ```
+    git commit -am "Meaningful commit message"
```
Comment (kritikaparmar-programmer) [inline CONTRIBUTING.md:85]: change it to  -> git commit -m "Meaningful commit message"

**Review by kritikaparmar-programmer — CHANGES_REQUESTED** [review]

**Comment by kcoder63** [comment]
changes done

**Review by shubhigupta991 — COMMENTED** [review]
Add the issue it resolves

**Comment by kcoder63** [comment]
I did mention above

**Review by shubhigupta991 — APPROVED** [review]

**Pull request merged**